### ISSUE TITLE
using sub-page html, auditor pointer fails to point elements

### ISSUE THREAD

**Issue by webteknik** [squizlabs/HTML_CodeSniffer#145]
Hi,
First, thanks for this great project, it is very useful.
We want to check for issues only in the sub-page HTML. i.e `<div id="container">...</div>` with the following way:

`HTMLCSAuditor.run('WCAG2AA', document.getElementById("container").innerHTML, options);`

However, when we run Auditor with the elements HTML, it can find the problems successfully but fails to point the elements:

> Unable to point to the element associated with this issue.Unable to point to this element because it is hidden from view, or does not have a visual representation.

Also I think it breaks while contructing page because following error is printed to console while auditor is running:

> `ReferenceError: $ is not defined <script type="text/javascript" src="/scripts/jquery.easing.min.js"></script>`

We could not find the relevant documentation, so we may be using auditor in a wrong way, if so, could you please point us the correct way. Thanks.

**Comment by ironikart** [comment]
Hi @webteknik 

The way you are using the auditor is passing a string of HTML so all context to the DOM is lost. You can adjust your code to the following which should work:

```javascript
HTMLCSAuditor.run('WCAG2AA', document.getElementById("container"), options);
```

This passes in the DOM Node object instead of the contents of the Node as a string so the auditor should be able to create pointers to the correct locations.
